File: GarageWizardAPI/GarageCoreMVC/Models/Job.cs
using GarageCoreMVC.Common;
using System.ComponentModel.DataAnnotations;

namespace GarageCoreMVC.Models
{
    public class Job
    {
        [Key]
        public string ID { get; set; } = Guid.NewGuid().ToString();

        [Required(ErrorMessage = DefaultValues.JobStringEmptyMessage)]
        [RegularExpression(@"^[a-zA-Z][a-zA-Z0-9- ]*$", ErrorMessage = DefaultValues.JobStringRegexMessage)]
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty ;

        [Required(ErrorMessage = DefaultValues.JobChargeableMessage)]
        public int Charge {  get; set; }
        public double Hours { get; set; }
        public string JobImagePath { get; set; } = "";
    }
}

File: GarageWizardClient/src/app/models/job.ts
export class Job
{
    id!: string;
    name!: string;
    description!: string;
    charge!: number;
    hours!: number;
    jobImagePath!: string;

    public constructor(jname: string, des: string, jcharge: number, jhours: number, jpath: string){
        this.name = jname;
        this.description = des;
        this.charge = jcharge;
        this.hours = jhours;
        this.jobImagePath = jpath;
    }
}